### PR TITLE
car store project

### PR DESCRIPTION
**Pull request by haroldmud**
### In this branch I did the following: :point_down: 
- Retrieved the data from the API and stored it in the Redux store.
- Filtered the retrieved data from the API using a Filter stateless component.
- Implemented a unique route for all the pages
- Created unit tests for pure functions in the Redux code.
- Deployed the project using Netlify

### CODE REVIEW

File: src/components/Model.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
 import React, { useEffect, useState } from 'react';
```
Comment (medaminedev66) [inline src/components/Model.js:1]: - [x] Following the project requirements, your app should have integration tests using React Testing Library to simulate user interaction. Check out this [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md) to learn how to do that.

File: src/components/styles/manufacture.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .home {
   border: purple 4px solid;
   text-decoration-line: none;
+  color: cadetblue;
 }
```
Comment (medaminedev66) [inline src/components/styles/manufacture.css:22]: - [x] Following the project requirements, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  - Typography: font face, size, and weight
  - Layout: composition and space between elements.
- Your home page design:
image

- The provided design:
image

- Your detail page design:
image

- The provided design:
image

File: README.md [hunk 3]
```diff
@@ -8,9 +8,13 @@
...
+## Live Demo 
 
-<!-- [Live Demo Link](https://livedemo.com) -->
+[Live Demo Link](https://deploy-preview-1--car-store-h.netlify.app/)
+
+## Presentation video
+
+[My video](https://www.loom.com/share/8992ec8322734c41a42f34a6f6f06abf)
 
 ## Getting Started
```
Comment (medaminedev66) [inline README.md:20]: [OPTIONAL] The `MIT link` in the README file is broken. I would recommend fixing that to make README more professional.

File: README.md [hunk 4]
```diff
@@ -8,9 +8,13 @@
 - Redux
```
Comment (medaminedev66) [inline README.md:8]: [OPTIONAL] It would be good to include a project description in your Readme, explaining what your project is about.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @haroldmud  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional
- Your project has a great structure

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @haroldmud 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**